### ISSUE TITLE
TiKV can't  read and write due to use rawclient and txnclient at the same time

### ISSUE THREAD

**Issue by jaime0815** [tikv/tikv#16284]
### What version of TiKV are you using?
TiKV 7.1
GO SDK: 2.1.4
### Steps to reproduce
No idea,   It worked well for reading and writing Tikv with different clients ago two months

### What did happened?
The scan failed due to the range key not being decoded,  I guess the key range overlaps between raw client keys and txn client keys.
```
errorVerbose="failed to decode region range key, key: \"3536306564663538306439663838632F6B762F6769642F74696D657374616D70\", err: invalid marker byte, group bytes \"b12/b2-86\"\n(1)
```

**Comment by guoxiangCN** [comment]
if you want use rawclient and txnclient at the same time, i think you should enable APIV2 in tikv, client-go also enable APIV2

**Comment by tonyxuqqi** [comment]
Yes,  API v2 has to be used for supporting using both raw tikv and tx tikv at same time.
